packages/universal-router-sdk/src/entities/protocols/pancakeswap.ts

```diff
@@ -215,8 +215,12 @@ async function addMixedSwap(
 
   for (const route of trade.routes) {
     const { inputAmount, outputAmount, pools } = route
-    const amountIn: bigint = SmartRouter.maximumAmountIn(trade, options.slippageTolerance).quotient
-    const amountOut: bigint = SmartRouter.minimumAmountOut(trade, options.slippageTolerance).quotient
+    const amountIn: bigint = SmartRouter.maximumAmountIn(trade, options.slippageTolerance, route.inputAmount).quotient
```

Comment:
Let's use amount defined in the previous line
```suggestion
    const amountIn: bigint = SmartRouter.maximumAmountIn(trade, options.slippageTolerance, inputAmount).quotient
```